wire-tests-swift/src/main/swift/ExternalMessage.swift

```diff
@@ -5,11 +5,18 @@ import Wire
 
 public struct ExternalMessage {
 
+    @Defaulted
```

Comment:
Assuming it's valid code, it'd be great to see `@Defaulted(defaultValue: Defaults.f)` in the generated code.

You _might_ need to have an explicit `init(defaultValue: T)` initializer?

Reply:
If you _are_ able to do that, you might not even need to set `self._f` in your init?

Reply:
I did do that initially but doing that causes problems with accessing `self` before its initialized.

Reply:
Bah